examples/http/server.rb

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'rubygems'
+require 'bundler/setup'
+require 'sinatra/base'
+# Require otel-ruby
+require 'opentelemetry'
+require 'opentelemetry/sdk'
+
+require_relative '../exporters/console'
+
+SDK = OpenTelemetry::SDK
+OpenTelemetry.tracer_factory = SDK::Trace::TracerFactory.new
+
+class App < Sinatra::Base
+  set :bind, '0.0.0.0'
+
+  configure do
+    # Configure tracer
+    exporter = Examples::Exporters::Console.new
+    processor = SDK::Trace::Export::SimpleSpanProcessor.new(exporter)
+    tracer = OpenTelemetry.tracer_factory.tracer('http.server', 'semver:1.0')
+    tracer.add_span_processor(processor)
+
+    set :tracer, tracer
+  end
+
+  get '/hello' do
+    # If route is available, use it:
+    span_name = env['sinatra.route'].split.last
+    # Otherwise, use RFC 3986 URI path value:
+    # span_name = request.path
+
+    settings.tracer.in_span(span_name, kind: 'Server') do |span|
```

Comment:
`OpenTelemetry::Trace::SpanKind::SERVER`

Reply:
_Note: we can use the symbol after #134_